mlflow/store/artifact/runs_artifact_repo.py

```diff
@@ -29,10 +29,10 @@ class RunsArtifactRepository(ArtifactRepository):
     users should take special care when constructing the URI.
     """
 
-    def __init__(self, artifact_uri):
+    def __init__(self, artifact_uri: str, tracking_uri: Optional[str] = None) -> None:
```

Comment:
To fix #16412, `RunsArtifactRepository` needs to be updated but I won't in this PR.